docs/content/image-repository.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Image Repository"
+---
+
+Override the container image repository used when pulling Kubernetes images.
+
+To enable the API server certificate SANs enable the `extraapiservercertsansvars` and `extraapiservercertsanspatch`
+external patches on `ClusterClass`.
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: ClusterClass
+metadata:
+  name: <NAME>
+spec:
+  patches:
+    - name: apiserver-cert-sans
```

Comment:
```suggestion
    - name: image-repository
```

Reply:
You're seeing how the sausage is made by most copy and pasting for new handlers, we would really benefit from a kubebuilder style generator.